src/models/hooks/goal.test.js

```diff
@@ -178,12 +178,15 @@ describe('goal hooks', () => {
           grantId: grant.id,
         },
         force: true,
+        paranoid: true,
```

Comment:
why have both flags?

Reply:
Unless I'm mistaken, ```force: true``` does a true delete but it does not ignore scopes (if a goal was already soft-deleted, ```force: true``` doesn't do anything)